shared/src/types/admin/Recipient.ts

```diff
@@ -21,7 +21,6 @@ export type Recipient = {
 	email: string;
 	first_name: string;
 	gender: string;
-	updated_on: Date;
```

Comment:
@ssandino is it okay to delete this? (there will be the last_updated_at field on all collections now)

Reply:
Yes, this field was a simple workaround to see if something changed, but no longer necessary!